stdlib/public/core/SIMDVector.swift

```diff
@@ -843,6 +874,13 @@ extension SIMD where Scalar: FloatingPoint {
   
   /// Returns the sum of the scalars in the vector.
   @_alwaysEmitIntoClient
```

Comment:
Need to look into this, but I think this attribute is what is preventing me from differentiating the sum()